### PR TITLE
Added more links in README, and fixed wording in a few places

### PR DESCRIPTION
**Pull request by kenerwin88**
*Description of changes:*
This PR is just some very minor changes to the README.md.  I added more links so that it's easy to get information on each of the things used in the repo, and tweaked some of the wording a bit (very minor).

### CODE REVIEW

**Comment by kenerwin88** [comment]
@vara-bonthu - This is mostly just to add some more links within the README.  Thanks for the great repo!

**Comment by vara-bonthu** [comment]
Thanks @kenerwin88 for your contribution on README.  I have added a new Bottlerocket feature with updated REDAME. I think this has caused some conflicts to your PR. If you could resolve the conflicts and resubmit the PR then i can review and merge it accordingly.

**Comment by kenerwin88** [comment]
Fixed @vara-bonthu, thank you for looking at the PR!  :)

File: README.md [hunk 1]
```diff
@@ -87,8 +92,8 @@ This modules ships the Fargate Continaer logs to CloudWatch
 
 # How to Deploy
 
-## Pre-requisites:
-Ensure that you installed the following tools in your Mac or Windows Laptop before start working with this module and run Terraform Plan and Apply
+## Prerequisites:
+Ensure that you have installed the following tools in your Mac or Windows Laptop before start working with this module and run Terraform Plan and Appl
```
Comment (vara-bonthu) [inline README.md:117]: Typo at the end. Should this be "Apply"?

**Review by vara-bonthu — CHANGES_REQUESTED** [review]
It looks good to me except one typo which needs fixing before the merge.

File: README.md (hunk 1 above)
Comment (kenerwin88) [reply README.md]: Oops, sorry about that, fixed!

**Review by vara-bonthu — APPROVED** [review]

**Pull request merged**